cloudbuild.yaml

```diff
@@ -1,25 +0,0 @@
-steps:
```

Comment:
We'll need to turn off the Cloud Build Trigger and disable the PR check requirement in GitHub settings

Reply:
Right after merge